pkg/internal/controller/controller_test.go

```diff
@@ -200,27 +200,40 @@ var _ = Describe("controller", func() {
 			<-sourceSynced
 		})
 
-		It("should not error when cache sync timeout is of sufficiently high", func() {
-			ctrl.CacheSyncTimeout = 1 * time.Second
+		It("should error when Start() is blocking forever", func() {
+			ctrl.CacheSyncTimeout = 0
```

Comment:
Should we maybe use something slightly higher? 

I think if we use 0 this test could also simply work because the timeout occurs before the go routine had a chance to run